README.md

```diff
@@ -216,11 +214,7 @@ final result = await barcodeScanner.processImage(inputImage);
 final text = await textDetector.processImage(inputImage);
 ```
 
-**To know more see this [example](./example/lib/VisionDetectorViews/text_detector_view.dart)**
-
-## Face Detection
```

Comment:
@sbis04: you deleted the Face Detection section I added.

Reply:
I am not sure how did this happen, maybe while merging it occurred. I am sending a PR with it re-added.